components/Bookmarks/Page.js

```diff
@@ -108,9 +108,7 @@ const Page = ({ t, me }) => {
                 {
                   feedLink: (
                     <Link route='feed' key='link'>
```

Comment:
Needs `passHref` when it's not a native a tag:
https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag